packages/chain-adapters/src/ethereum/EthereumChainAdapter.test.ts

```diff
@@ -420,6 +420,47 @@ describe('EthereumChainAdapter', () => {
     })
   })
 
+  describe('signMessage', () => {
+    it('should sign a properly formatted signMessageInput object', async () => {
+      const args = makeChainAdapterArgs()
+      const adapter = new ethereum.ChainAdapter(args)
+      const wallet = await getWallet()
+      wallet.ethSignMessage = async () => ({
```

Comment:
we should be able to call the native wallet directly without overwriting this function since it doesn't require any interaction with the blockchain